temporary fix because of alpha, has to be dropped as soon as gcc's bug is fixed.

--- a/src/opengl/opengl.pro
+++ b/src/opengl/opengl.pro
@@ -5,6 +5,7 @@
 win32-msvc*|win32-icc:QMAKE_LFLAGS += /BASE:0x63000000
 solaris-cc*:QMAKE_CXXFLAGS_RELEASE -= -O2
 
+QMAKE_CXXFLAGS_HIDESYMS -= -fvisibility-inlines-hidden
 
 include(../qbase.pri)
 
